molecule/default/tests/test_default.py

```diff
@@ -1,20 +1,60 @@
 import os
 import re
-
+import pytest
 import testinfra.utils.ansible_runner
 
+###############################################################
+# ASC-187: Verify nova_force_config_drive setting is disabled  #
+################################################################
```

Comment:
nit pick: the # signs at the end of this comment block do not align.

Reply:
Will fix in the next push